examples/custom_vocabularies.js

```diff
@@ -0,0 +1,43 @@
+const revai = require('revai-node-sdk');
+const token = require('./config/config.json').access_token;
+
+(async () => {
+    // Initialize your client with your revai access token
+    var client = new revai.RevAiCustomVocabulariesClient(token);
+
+    cv_job = client.submitCustomVocabularies([{
+            phrases: [
+                "add",
+                "custom",
+                "vocabularies",
+                "here"
+            ]
+        }])
+
+    console.log(`Job Id: ${cv_job.id}`);
+    console.log(`Status: ${cv_job.status}`);
+    console.log(`Created On: ${cv_job.created_on}`);
+
+    /**
+     * Waits 5 seconds between each status check to see if job is complete.
+     * note: polling for job status is not recommended in a non-testing environment.
```

Comment:
Another place where we aren't consistent with job vs custom vocab terms

Reply:
addressed